src/components/element-editor/validation/validation-child.tsx

```diff
@@ -0,0 +1,92 @@
+import { Form } from "@lifesg/react-design-system";
```

Comment:
use subdirectory import to reduce bundled size

```suggestion
import { Form } from "@lifesg/react-design-system/form";
```